### ISSUE TITLE
Support reading api key from an env var

### ISSUE THREAD

**Issue by junjizhi** [pablo-co/bamboo_postmark#25]
Support the config like:
```
config :my_app, MyApp.Mailer,
      adapter: Bamboo.PostmarkAdapter,
      api_key: {:system, "API_KEY"}
```
This is allowed in Sendgrid adapter.

**Comment by pablo-co** [comment]
Thank you for the contribution! I will be adding the corresponding tests to the PR. Will create a new release during the weekend!

**Comment by pablo-co** [comment]
Closed by #26
